x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/_layer_panel.scss

```diff
@@ -27,6 +27,14 @@
   overflow: hidden;
 }
 
+.lnsLayerPanel__dimension-hidden {
+  visibility: hidden;
+}
+
+.lnsLayerPanel__dimension-replace {
+  text-decoration: line-through;
+}
+
```

Comment:
To avoid the effect of 'x' button being slower than the rest when dragging, you could add a custom className and overwrite `transition: all 250ms ease-in` setting coming from EUI, eg:
```
    transition: all 250ms ease-out, visibility 0ms;
```

Here's what I mean:
![image](https://user-images.githubusercontent.com/4283304/91275947-1fe32a00-e781-11ea-9fe5-0cce74f5fd0f.png)